### PR TITLE
Create CONTRIBUTING.adoc

### PR DESCRIPTION
**Pull request by lauracowen**
Ellen: Are you happy with the instructions?
Alasdair: Are you happy with the CONTRIBUTING file being added to this repo?

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
+= Contributing a blog post to Open Liberty
```
Comment (ellenwyllie) [inline CONTRIBUTING.adoc]: Maybe the title should be more specific and say something about signing the PRs. This title is very similar to the title of the README so I was initially confused about the purpose of this file

File: README.md [hunk 2]
```diff
@@ -107,19 +123,5 @@ You will see `Jekyll` detect your new files and regenerate the blog files.  You
             ...done in 121.8705398 seconds.
 ```
 
-### Restarting the container
```
Comment (ellenwyllie) [inline README.md]: Is there a reason we want to remove this section about restarting the container?

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+If you you'd like to write a blog post for OpenLiberty.io, let us know! See [README](README.md) for instructions then see below.
```
Comment (ellenwyllie) [inline CONTRIBUTING.adoc]: This link to the README doesn't appear to be working when I preview the page.

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (lauracowen) [reply CONTRIBUTING.adoc]: I was wondering about that. It's a standard file name that I think we should stick with for consistency. I think actually, we should move most of the instructions from the README to the CONTRIBUTING file. I'll have another look tomorrow.

**Review by NottyCode — APPROVED** [review]

**Comment by lauracowen** [comment]
Closing because I'm making the update again as part of documentation the wider process now we've changed the github workflow.

**Pull request closed without merging**